docs/cookbook/shop/cms.rst

```diff
@@ -0,0 +1,45 @@
+How to manage content in Sylius?
+=================================
+
+Why do you need content management system?
+--------------------------------------------
+
+Content management is one of the most important business aspects of modern eCommerce apps.
+Providing store updates like new blog pages, banners and promotion images is responsible for building the conversion rate
+either for new and existing clients.
+
+Content management in Sylius
+-----------------------------------
+
+Default Sylius standard app does not come with a content management system. Don’t worry - our community has taken care of it. As Sylius does have an awesome dev oriented plugin environment,
+our friends from `BitBag <https://bitbag.shop>`_ decided to develop their flexible CMS module. You can find it on `their GitHub <https://github.com/BitBagCommerce/SyliusCmsPlugin>`_.
+
+.. tip::
+
+    Check their `GitHub <https://github.com/BitBagCommerce>`_ for more awesome plugins like payment gateways integrations, shipment management and many more. Don't forget to leave a star if you liked their work!
```

Comment:
All the plugins are already listed on this page: http://docs.sylius.org/en/latest/plugins/index.html#the-official-list-of-plugins No need to mention it here. Cookbooks are focusing on solving certain problems, plugin advertisements are rather clutter here.

Reply:
Yeah, I feel a little bit overwhelmed with amount of BitBag links in this cookbook 🎉

Reply:
👌